### DISCUSSION TITLE
Replit Bot Not Working

### DISCUSSION THREAD

**Discussion by daaanii** [PrismarineJS/mineflayer#2260] [Q&A] [answered]
I have a mineflayer bot, that uses as account a premium one, with auth type set to microsoft.
When I run the program on my pc, it works fine, but when i try to run it on replit it doesn't  work and throws an invalid credentials error, generated from the xbox live module. Does replit block the connection to the xbox's login servers?

**Comment by u9g** [comment]
It shouldn’t

**Comment by AsherJingkongChen** [comment, marked as answer]
you can move ur acc to a microsoft family group.
I succeeded
